RELEASE/scripts/sl_ascend/sl_util.ash

```diff
@@ -2834,6 +2842,14 @@ boolean providePlusNonCombat(int amt, boolean doEquips)
 		removeCombat();
 	}
 
+        if((numeric_modifier("Combat Rate").to_int() > amt)
+           && get_property("horseryAvailable").to_boolean()
+           && (get_property("_horsery") != "dark horse")
+           && (my_meat() >= 1000))
+        {
+                cli_execute("horsery dark");
```

Comment:
I don't understand how this doesn't contend with the horsery setting LX_craftAcquireItems() until you run out of meat.
```
if(sl_my_path() != "Community Service")
	{
		getHorse("noncombat");
```

Maybe use something parallel to the Dark Gyffte form solution? (instead of changing the horse immediately, set a local preference, check in the preadventure or in ccAdv whether we're in the desired state, if so fix it if possible).

Reply:
I was watching to see if my meat would disappear in the span of a few seconds after putting this in, haha. At the very least, I can confirm it didn't spend all my meat, cause I have more than I started the day with.

But it's probably spending more than necessary. Your alternate approach will probably fix that, so I'll make that switch rather than dig through my session logs.

Reply:
I *think* it's spending 1K meat a turn (the cost of two switches), but I'm not sure of that.